### PR TITLE
summarization feedback

### PR DESCRIPTION
**Pull request by rshih32**
### Regular Summarization

Screen Shot 2023-09-25 at 12 35 06 PM

Screen Shot 2023-09-25 at 12 35 02 PM

### Tangential/Related Work Summarization

Screen Shot 2023-09-25 at 12 36 18 PM

Screen Shot 2023-09-25 at 12 34 50 PM

### CODE REVIEW

File: trulens_eval/trulens_eval/feedback/prompts.py [hunk 1]
```diff
@@ -219,6 +219,56 @@
 {response}
 """
 
+"""
```
Comment (joshreini1) [inline trulens_eval/trulens_eval/feedback/prompts.py:222]: This prompt seems to be missing its variable assignment?

File: trulens_eval/trulens_eval/feedback/provider/openai.py [hunk 2]
```diff
@@ -1214,6 +1214,34 @@ def _get_answer_agreement(
...
+        ```
+        The `on_input_output()` selector can be changed. See [Feedback Function Guide](https://www.trulens.org/trulens_eval/feedback_function_guide/)
+
+
+        Args:
+            source (str): Text corresponding to source material. 
+            summary (str): Text corresponding to a summary.
+
+        Returns:
+            float: A value between 0 and 1. 0 being "main points missed" and 1 being "nno main points missed".
```
Comment (joshreini1) [inline trulens_eval/trulens_eval/feedback/provider/openai.py:1238]: nno -> no

**Review by joshreini1 — CHANGES_REQUESTED** [review]

File: trulens_eval/trulens_eval/feedback/prompts.py (hunk 1 above)
Comment (rshih32) [reply trulens_eval/trulens_eval/feedback/prompts.py]: thanks; that was an old one I was experimenting with. Removed it

**Review by joshreini1 — APPROVED** [review]

File: trulens_eval/trulens_eval/feedback/prompts.py [hunk 3]
```diff
@@ -219,6 +219,40 @@
 {response}
 """
 
+SUMMARIZATION_PROMPT = """
+You are a helper for summarization tasks. You will do two steps. You must both steps, and most importantly, you must add a score.
```
Comment (joshreini1) [inline trulens_eval/trulens_eval/feedback/prompts.py:223]: must *do* both steps

**Pull request merged**